src/conversion.rs

```diff
@@ -200,6 +200,31 @@ pub trait IntoPyObject<'py>: Sized {
 
     /// Performs the conversion.
     fn into_pyobject(self, py: Python<'py>) -> Result<Self::Output, Self::Error>;
+
+    /// Converts slice of self into a Python object
+    fn iter_into_pyobject<I, E>(
```

Comment:
Should we make this `#[doc(hidden)]`?

Reply:
I think that makes sense, I will add that.